src/part.ts

```diff
@@ -4,7 +4,16 @@ export interface Element {
   direction: number;
 }
 
+export interface Label {
```

Comment:
the labels can go in circular or in circular/labels. Are created from parts' annotations but not associated with the parts themselves